salon_creator/lib/firebase/database.dart

```diff
@@ -8,10 +9,25 @@ final salonsRef = FirebaseFirestore.instance.collection('salons').withConverter(
       toFirestore: (salon, _) => salon.toMap(),
     );
 
-void addSalonsToDatabase(Salon salon) async {
-  try {
-    await salonsRef.add(salon);
-  } on FirebaseException catch (e) {
-    print(e.message);
+class DbHandler {
+  DbHandler();
+  addSalon(Salon salon) async {
+    try {
+      await salonsRef.add(salon);
+    } on FirebaseException catch (e) {
+      print(e.message);
```

Comment:
I think this will silence the exceptions. Is there a way for the caller of this function to know that the salon creation failed? I think a better approach would be to not catch the exception here, but throw it to the caller and the caller can catch it and make a decision about what to do when such an error happens.